fcrepo-service-activemq/pom.xml

```diff
@@ -76,6 +76,11 @@
     </dependency>
 
     <dependency>
+      <groupId>xerces</groupId>
```

Comment:
How does this dependency addition influence logging?  Just curious,  Otherwise; beautiful